script/test

```diff
@@ -283,6 +283,7 @@ do_build_otbr_docker()
     (
         ./script/git-tool clone https://github.com/openthread/ot-br-posix.git --depth 1 "${otbrdir}"
         cd "${otbrdir}"
+        sed -i '/^#define MBEDTLS_SSL_EXPORT_KEYS/a #define MBEDTLS_SSL_KEEP_PEER_CERTIFICATE' third_party/openthread/mbedtls-config.h
```

Comment:
This needs to be removed after https://github.com/openthread/ot-br-posix/pull/827 is merged.

Reply:
Reverted.